app/routes/missions/hawc.md

```diff
@@ -0,0 +1,33 @@
+---
+meta:
+  title: GCN - HAWC
+---
+
+# High Altitude Water Cherenkov Gamma-Ray Observatory
+
+<img 
+  src="/_static/img/hawc-logo.png"
+  width="200"
+  align="right"
+  alt="HAWC logo"
+/>
+
+**Inauguration:** March 2015
```

Comment:
"Inauguration" sounds too grand. How about:

```suggestion
**Start of Operations:** March 2015
```

Reply:
How about "Start of full operation"? Inauguration was a typical term/date that has been used in HAWC publications, and technically there is data going back to 2012 when the observatory was only 10% built. HAWC has a modular design that allows for partial array operation. Normally only data after full operation is of interest, but for transients even partial sensitivity may be of interest if there is coverage.

Reply:
I like it.

Reply:
OK, I pushed the change. Still iffy on how the web interface vs command line works. I was a SVN person (CVS if going even further back).

Reply:
💯 Nice work, and thanks for your first PR!